### ISSUE TITLE
Bug - disable outline on target of skip link

### ISSUE THREAD

**Issue by tessereth** [AusDTO/gov-au-ui-kit#300]
## What I did

We currently have a "skip to main content" link at the top of our page but theoretically it requires the main content to be focusable by adding `tabindex="-1"` to the target of the skip link. Unfortunately this adds a focus boarder around the main content when you click on it. Previously we've solved this problem by removing the tabindex attribute and the skip link still seems to work (I've tested in latest chrome and IE9) but the axe-core accessibility test for this fails without the tabindex. I'm currently solving this using the following CSS (where `#content` is the id for the target of the skip link) but I'm interested what you thought about solving this in ui-kit.

```
#content:focus {
  outline: none;
}
```
### What browser and device I was using

Chrome
## What I expected to see

No outline
## What I saw

![screen shot 2016-08-24 at 10 59 24 am](https://cloud.githubusercontent.com/assets/4583474/17914969/df9be630-69e9-11e6-9afb-cfaf652f09fc.png)

**Comment by TrebBrennan** [comment]
Notes:
Confirmed with @tessereth what the `tabindex="-1"` on a div is intended to do, as most browsers should preform the skip-link as expected without having to apply direct focus to the div ([This bug](https://bugs.chromium.org/p/chromium/issues/detail?id=37721) appears to be fixed in Chrome now)  ....  It turns out this was added in response to an accessibility suggestion from [axe-core](https://github.com/dequelabs/axe-core)

Question:
Is there a need (from axe-core) to add `tabindex` for other elements too? For example the `.index-links` tend to link to Heading elements which don't naturally receive focus either.

Accessibility:
Should an element receive focus without an indication that it has focus?

**Comment by TrebBrennan** [comment]
Tested skip-links and they do not behave as intended in all versions of IE (6-Edge: pages scrolls but focus does not change position).

We will likely ship an out-of-the-box JS solution to this bug in the future so users of the Kit don't need to think about it...

@tessereth For the time being your solution to continue using `tabindex="-1"` and a `outline: none;` is the best way to go. Thanks muchly for picking this one up.

**Comment by klepas** [comment]
Tracked under `DESIGN-394`.

**Comment by klepas** [comment]
Other than the CSS fix Tessa noted above, I’m not sure how to best proceed on this one. Thoughts team?

**Comment by hannah-ustwo** [comment]
Possibly need more testing to address if it is the right solution. 
This won't be a priority for the Guides team in sprint 40.
